docs/CODING_STYLE.rst

```diff
@@ -67,32 +67,198 @@ Coding guide
 
 Comments
 --------
+Above every function prototype in ``.h`` files that export the public API,
+add a Doxygen-formatted comment like this:
 
-Before every function have in ``.h`` files a comment like this:
-
-.. code:: c
+.. code-block:: c
 
    /**
-    * Return with the screen of an object
-    * @param obj pointer to an object
-    * @return pointer to a screen
+    * Create and add keyboard, mouse and scroll wheel objects and connect them to x11 display.
+    *
+    * This is a convenience method handling the typical input initialisation of an X11 window:
+    * - create keyboard (lv_x11_keyboard_create)
+    * - create mouse (with scroll wheel, lv_x11_mouse_create lv_x11_mousewheel_create)
+    *
+    * @param[in]  disp       object created by lv_x11_window_create()
+    * @param[in]  mouse_img  optional image description for mouse cursor
+    *                            (NULL for no/invisible mouse cursor)
+    *
+    * @note  Include any subtle points an API user or maintainer would need to know.
     */
-   lv_obj_t * lv_obj_get_screen(lv_obj_t * obj);
+   void lv_x11_inputs_create(lv_display_t * disp, lv_image_dsc_t const * mouse_img);
+
+Separate sections within the comment with a blank line for added readability.
```

Comment:
Not used now either.

Reply:
@kisvegabor Hi, Gabor!  Ouch.  Okay, I removed that line (to be re-pushed shortly).  Kevin and I discussed this in issue #6480, and I think I erroneously interpreted no response from you on that point as agreement.  While I am coding, I rely on documentation in the code more than in an external documentation, so I find it extremely helpful when the documentation in the code is easier to read.  Are you thinking of approach this gradually?  Or prefer no blank lines?  What are your thoughts are about it for now and in the future?

Reply:
I'd prefer no blank lines to keep the docs section more condensed.

Reply:
> I'd prefer no blank lines to keep the docs section more condensed.

Understood.  Thank you for the clarification.